src/components/Homes.js

```diff
@@ -0,0 +1,146 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faMapMarkerAlt, faCloud, faWind, faSun, faThermometerHalf, faInfoCircle,
+} from '@fortawesome/free-solid-svg-icons';
+import { fetchCities } from '../redux/home/homeSlice';
+import './Homes.css';
+
+function Homes() {
+  const dispatch = useDispatch();
+  const { data, status, error } = useSelector((state) => state.home);
+
+  useEffect(() => {
+    dispatch(fetchCities());
+  }, [dispatch]);
+
+  if (status === 'loading') {
+    return <div>Loading...</div>;
+  }
+
+  if (status === 'failed') {
+    return (
+      <div>
+        Error:
+        {error}
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <div>
+        <Link to="/about"><FontAwesomeIcon icon={faInfoCircle} style={{ fontSize: '40px', marginBottom: '-30px', marginLeft: '10px' }} /></Link>
+        <h1 className="heading">Abuja, Nigeria</h1>
+        <p className="cat">{`${((Math.floor(data.timezone / 3600)) >= 0 ? '+' : '-') + (Math.abs(Math.floor(data.timezone / 3600))).toString().padStart(2, '0')}.${Math.abs(Math.floor(data.timezone % (3600 / 60))).toString().padStart(2, '0')} UTC`}</p>
+      </div>
+      <p className="stats">STATS BY CATEGORY</p>
+      <div className="home-items">
+        <Link to="/coordinates">
+          {data.coord && (
+          <p className="home-item">
+            <FontAwesomeIcon
+              icon={faMapMarkerAlt}
+              style={{
+                color: 'rgb(53, 53, 165)', fontSize: '60px', marginBottom: '-80px', paddingTop: '15px',
+              }}
+            />
+            <span className="item-wrap">
+              Coordinates
+              <span className="item">
+                {Object.keys(data.coord).length}
```

Comment:
From the project's requirement, It was mentioned that you should add another feature where you use a Filter stateless component to filter data retrieved from the API.

e.g You can implement that by adding a search bar where a user can type for example a category name he wants to get information from. I know you can do that 💪

![image](https://user-images.githubusercontent.com/78251927/225779918-4e774ba8-b4a4-4f53-b154-d7f43f07d952.png)